### PR TITLE
feat(repeatString): add instruction to use loops for implementation rather than builtin method

### PR DESCRIPTION
**Pull request by nikitarevenco**
There is a builtin method for accomplishing the same outcome, so I have updated the instructions as the point of the exercise is to practice loops.

### CODE REVIEW

**Comment by MaoShizhong** [comment]
I'm hesitant on this one.

I believe the intention of the exercise is to get learners to get more practice with using loops. The hint quite firmly suggests this, but I feel for the exercise's intent, if loops are what it wants the learner to practise, it would be better placed in the actual exercise instructions instead of the hints section.

Basically saying that even though there is a built in string method for repeating strings, we want to practise loops so do it manually with loops instead. That way, they're also made aware of the string method.

**Comment by nikitarevenco** [comment]
> I'm hesitant on this one.
> 
> I believe the intention of the exercise is to get learners to get more practice with using loops. The hint quite firmly suggests this, but I feel for the exercise's intent, if loops are what it wants the learner to practise, it would be better placed in the actual exercise instructions instead of the hints section.
> 
> Basically saying that even though there is a built in string method for repeating strings, we want to practise loops so do it manually with loops instead. That way, they're also made aware of the string method.

I've repurposed this PR

File: 02_repeatString/README.md
Comment (MaoShizhong) [inline 02_repeatString/README.md:1]: Thanks for this. Mind removing the "You're going to want to use a loop for this one." hint as it's now redundant? The rest can be left as is.

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

File: 02_repeatString/README.md
Comment (nikitarevenco) [reply 02_repeatString/README.md:1]: Ok, I removed it

**Review by MaoShizhong — APPROVED** [review]

**Pull request merged**